src/others/components/CollapsibleList.tsx

```diff
@@ -25,10 +33,29 @@ export const CollapsibleTable = ({ rows, renderRowData, ...tableProps }: Collaps
     setDisplayedRows([...displayedRows, ...nextRows]);
   };
 
+  const toggleZoomCity = (coordinates: Coordinates) => {
+    const { latitude, longitude } = coordinates;
+    if (selectedCity?.latitude === latitude && selectedCity?.longitude === longitude) {
+      setSelectedCity(undefined);
+    } else {
+      setSelectedCity(coordinates);
+    }
+  };
+
   useEffect(() => {
     setDisplayedRows(getGirstBatch(rows));
   }, [rows]);
 
+  useEffect(() => {
+    if (selectedCity) {
+      const { latitude, longitude } = selectedCity;
+      map.default?.flyTo({ center: [longitude, latitude], duration: 2000, zoom: 10 });
+    } else {
+      // Reset to initial map view
+      map.default?.flyTo({ center: [30.5240501, 48.4501071], duration: 2000, zoom: 5 });
```

Comment:
ideally we should not hardcode this in two places, but could be fixed in a followup

Reply:
agreed. it didn't feel right to export it from `Map.tsx`, although maybe it's fine for now. AFAICT we don't have any shared `constants.ts` file yet.

Reply:
we don't, but feel free to create one. it probably makes sense to have it anyway